### DISCUSSION TITLE
Where the frontend VEP_SV_TYPES data come from?

### DISCUSSION THREAD

**Discussion by SimonLi5601** [broadinstitute/seqr#2772] [Q&A] [answered]
In the hail-elasticsearch-pipelines, the sv_pipeline script prepares the [SVTYPE_DETAILS](https://github.com/broadinstitute/seqr/blob/e0c179c36c0f68c892017de5eab2e4c1b9ffdc92/ui/shared/utils/constants.js#L893), but there are [VEP_SV_TYPES](https://github.com/broadinstitute/seqr/blob/e0c179c36c0f68c892017de5eab2e4c1b9ffdc92/ui/shared/utils/constants.js#L530), from the naming imply VEP tools?

Thanks!

**Comment by hanars** [comment, marked as answer]
Sorry, the variable name is confusing. There is no VEP for SVs. The constant is just used for displaying the annotations and is included in the `ORDERED_VEP_CONSEQUENCES` constant that primarily has actual SNP VEP consequences so that was the name it ended up with
